gcloud-java-cloudtrace/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>gcloud-java-cloudtrace</artifactId>
```

Comment:
It's a holdover. FYI I think it's actually `Stackdriver Trace` - see https://cloud.google.com/trace/ . I can remove the 'cloud' part and just call it Trace.